ovirt_imageio/client/_upload.py

```diff
@@ -22,7 +22,7 @@
 
 DiskInfo = namedtuple(
     "DiskInfo",
-    "name,initial_size,provisioned_size,content_type,format,sparse")
+    "name,initial_size,provisioned_size,content_type,format,sparse,is_zero")
```

Comment:
An idea not for this PR: Could this `namedtuple` maybe benefit of being wrapped into a class?

For example, `is_zero` does not need to be set, can be a class property:
```python
    @property
    def is_zero(self):
        return (self.format == FORMAT_QCOW2) or self.sparse
```

All attributes could be stored as strings, and transformed into `sdk` types in `@property` getters, so we have a central point where this transformation is done.

Reply:
We can but we need to document why. We can move the comment to the property, but it means that the comment related to prepare are spread in different locations in the file. It better that all the logic is in one place.

Reply:
Just an idea, since this new attribute is a derivative of other attributes, I thought it may fit as a `property`. And then the I wondered what other logic could be moved to the class, to justify having it.

Let's keep it in a box for now, for the time being is better to have the logic together, yes.